port_ocean/config/base.py

```diff
@@ -17,7 +17,7 @@
 def read_yaml_config_settings_source(
     settings: "BaseOceanSettings", base_path: str
 ) -> dict[str, Any]:
-    yaml_file = getattr(settings.__config__, "yaml_file", "")
+    yaml_file = getattr(settings.Config, "yaml_file", "")
 
```

Comment:
seems like the `base_path` is redundant now, and you can use `settings._bash_path` inside the method